src/main/java/Calculater.java

```diff
@@ -0,0 +1,166 @@
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class Calculater {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4535407)